test/src/util/pelz_json_parser_test_suite.c

```diff
@@ -447,21 +209,13 @@ void test_request_decoder(void)
     free_charbuf(&data);
 
     cJSON_DeleteItemFromObject(json_dec_signed, "dec_data");
-    cJSON_DeleteItemFromObject(json_dec_signed, "dec_data_len");
     cJSON_DeleteItemFromObject(json_enc_signed, "enc_data");
```

Comment:
These will have to be fixed with just data instead of ```enc_data``` and ```dec_data```.

Reply:
Fixed with 4d59ae2.